docs/zero.block

```diff
@@ -19,8 +19,57 @@
                     "peer_key": "292a8714694095edce6be799398ed5d6244cd7be37eb813106b217d850d261f2"
                 },
                 {
+                    "command_type": "CreateRole",
```

Comment:
Genesis block looks more complicated and complicated, I guess, we need to change it with automatic generation. But ofc, this issue not related to current PR.